validator/src/main/resources/validations/java/ecs/trace-validation.yml

```diff
@@ -0,0 +1,6 @@
+-
+  validationType: "trace"
+  callingType: "none"
+  httpMethod: "get"
+  httpPath: "/"
```

Comment:
Why do we need httpMethod/httpPath if we have `none` callingType?

Reply:
I found the httpMethod and httpPath is still being used [here](https://github.com/aws-observability/aws-application-signals-test-framework/blob/442f8fbab821d53222bc87e5e4e23c5e71d72d93/validator/src/main/java/com/amazon/aoc/validators/TraceValidator.java#L143) during trace validation even callingType is none.